agent/util-scripts/pbench-tool-meister-start

```diff
@@ -809,13 +1135,48 @@ if __name__ == "__main__":
         default=None,
         help="The list of system information items to be collected.",
     )
+    parser.add_argument(
+        "--orchestrate",
+        dest="orchestrate",
+        default="create",
+        choices=("create", "existing"),
+        help=(
+            "The `create` keyword directs the command to create the various"
```

Comment:
The message needs to mention what the default is: apparently `--help` does not say anything about the default value (see e.g. the top of test-54.txt).